### DISCUSSION TITLE
Admin add time limited free trial

### DISCUSSION THREAD

**Discussion by ideatrap** [Glench/ExtPay#170] [Ideas]
Request to add the following feature:
Feature: In subscription pricing, allow admin to add user for time limited free trial. For example, 1 year free usage. Currently, it's life time free usage.

Reason: It helps admin to market the product and add seed user. At the same time, give limited benefits to the seed user.

**Comment by Glench** [comment]
Nice idea! For now you can just add them as a free user and let me know when you'd like to revoke their free access.
